scripts/travisci/check_flake8.sh

```diff
@@ -0,0 +1,146 @@
+#!/usr/bin/env bash
+# Exit status of this script
+exit_status=0
+
+# Add an option to run the incremental verification locally to reduce the time
+# spent in travis for error verification. It's required to rebase the feature
+# branch with the master branch to avoid checking on unknown changes.
+while getopts ":l" opt; do
+  case ${opt} in
+    l)
+      TRAVIS_PULL_REQUEST="false"
```

Comment:
you can check the TRAVIS or CI variables instead:
https://docs.travis-ci.com/user/environment-variables/#Convenience-Variables

Reply:
Done